src/js/themes/vanilla.js

```diff
@@ -188,24 +188,24 @@ export default {
   heading: {
     level: {
       1: {
-        medium: { size: '48px', height: 1.125 },
-        small: { size: '24px', height: 1.333 },
-        large: { size: '96px', height: 1.125 },
+        medium: { size: '48px', height: 1.125, maxWidth: `${baseSpacing * 24}px` },
```

Comment:
I would expect these maxWidth values to be proportional to the font size. For instance, maxWidth for 48px font-size should be larger than maxWidth for 18px font-size.